packages/edit-site/src/components/site-hub/style.scss

```diff
@@ -36,8 +36,17 @@
 
 	&:hover,
 	&:focus,
-	&:active {
-		color: $gray-200;
+	&:active,
+	&:focus-visible {
+		svg {
+			opacity: 1;
+			transition: opacity 0.1s ease-in-out;
```

Comment:
I'd suggest to remove the transition here and add it to the normal state of the svg.